tests/Feature/Http/Controllers/RatingControllerTest.php

```diff
@@ -35,12 +35,13 @@ public function setUp(): void
     public function index_displays_view()
     {
         $ratings = Rating::factory()->count(3)->create();
+        $users = User::factory()->has(Rating::factory()->count(3))->count(3)->create();
```

Comment:
Factory accepts a count parameter, so you should be able to do: 
```suggestion
        $users = User::factory(3)->has(Rating::factory()->count(3))->create();
```
and get the same results